admin_guide/manage_nodes.adoc

```diff
@@ -477,13 +477,123 @@ In rare cases, a router pod may not have a host port configured. In those cases,
 it is important to follow the xref:infrastructure-nodes[recommended restart
 process] for infrastructure nodes.
 
+[[modifying-nodes]]
+== Modifying Nodes
+
+// tag::node-configmap[]
+During installation, {product-title} creates a configuration map in the *openshift-node* project for each type of node group: 
```

Comment:
configmaps is specific resource, we'd better update "configuration map" to "configmaps"